Sources/StreamChat/WebSocketClient/Events/EventType.swift

```diff
@@ -120,6 +122,7 @@ extension EventType {
         case .userBanned: return try UserBannedEventDTO(from: response)
         case .userUnbanned: return try UserUnbannedEventDTO(from: response)
         
+        case .channelCreated: throw ClientError.UnsupportedEventType()
```

Comment:
@polqf bit of weird/confusing approach, isn't better to parse the event correctly and simply do nothing about it when it's received?

Reply:
marking "channel.created" as an unsupported event is the strange bit, totally fine to do nothing when such event is received though

Reply:
The idea is not to have to maintain any kind of code for an event we don't even need. Otherwise both an Event and an EventDTO is required as per our contract.

Maybe the naming is what can be changed here. Might rename it to IgnoredEventType